src/plugins/initiatives/initiativeFile.js

```diff
@@ -19,6 +20,28 @@ export const INITIATIVE_FILE_SUBTYPE = "INITIATIVE_FILE";
  */
 export type InitiativeFile = InitiativeFileV020;
 
+/**
+ * Stored inside a initiatives.json db file with other initiatives.
+ * presently the `id` prop is used as the key inside the db file
+ */
+export type InitiativeFileV100 = {|
```

Comment:
This type shouldn't really be called `InitiativeFile` because it no longer is describing a file, right? We're dropping the assumption that there's a 1:1 mapping between initiatives and files. So this could be called something like `InitiativeEntry` instead, and there'd be an `InitiativeStore` type which would correspond to the `db.json`.